docs/devices.ipynb

```diff
@@ -0,0 +1,410 @@
+{
```

Comment:
<p>Great example. </p><p> </p><p>Nit: Commas after "By default," and "on all the moments,". </p><p> </p><p>typo (?) <code>cirq.GriDeviceMetadata</code>   -&gt; <code>cirq.GridDeviceMetadata</code>.</p><p> </p><p>What's the difference between using the two DeviceMetadata classes/why would you use one or the other?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5395/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
Fixed.